docs/networking/clusternetwork.md

```diff
@@ -31,14 +31,30 @@ Specifications including network devices of the Harvester hosts can be different
 
 Network configuration only works under a certain cluster network. Each network configuration corresponds to a set of hosts with uniform network specifications. Therefore, multiple network configurations are required for a cluster network on non-uniform hosts.
 
-### Network
+### VM Network
 
-A network is an interface in a virtual machine that connects to the host network. As with network configuration, every network except the built-in [management network](./harvester-network.md#management-network) must be under a cluster network.
+A VM network is an interface in a virtual machine that connects to the host network. As with network configuration, every network except the built-in [management network](./harvester-network.md#management-network) must be under a cluster network.
 
 Harvester supports adding multiple networks to one VM. If a network's cluster network is not enabled on some hosts, the VM that owns this network will not be scheduled to those hosts.
 
 Please refer to [network part](./harvester-network.md) for more details about networks.
 
+### Relationship Between Cluster Network, Network Config, VM Network
+The following diagram shows the relationship between cluster network, network config, and VM network.
+
+![](/img/v1.2/networking/relation.png)
+
+- Every network config and VM network is under a cluster network.
+- To categorize hosts based on network specifications, you can assign a label to each host and then add a network config for each group of hosts using a node selector. For ClusterNetwork-A in the diagram above, we group all hosts into 3 groups, respectively:
+   - The first group includes host0 and host2, corresponding to network-config-A
+   - The second group includes host1, corresponding to network-config-B
+   - The third group includes the remaining host3, host4 and host5, corresponding to no network config.
+- The cluster network will only be effective on hosts that are covered by the network configuration. If a VM uses a VM network under a specific cluster network, it can only be scheduled on a host where the cluster network is active.
+   In the diagram above, we can see that:
+   - ClusterNetwork-A is active on host0, host1 and host2. VM0 uses VM-network-A so it can be scheduled on host0, host1 or host2.
+   - VM1, which uses both VM-network-B and VM-network-C, can be only scheduled on host2 where both ClusterNetwork-A and ClusterNetwork-B are active.
+   - VM0, VM1 and VM2 cannot run on host3 where the two cluster networks are inactive.
```

Comment:
```suggestion
The cluster network is only effective on hosts that are covered by the network configuration. If a VM uses a `VM network` under a specific cluster network, it can only be scheduled on a host where the cluster network is active. In the diagram, we can see that:
   - `ClusterNetwork-A` is active on host0, host1, and host2. `VM0` uses `VM-network-A`, so it can be scheduled on any of these hosts.
   - `VM1` uses both `VM-network-B` and `VM-network-C`, so it can only be scheduled on host2 where both `ClusterNetwork-A` and `ClusterNetwork-B` are active.
   - `VM0`, `VM1`, and `VM2` cannot run on host3 where the two cluster networks are inactive.

Overall, this diagram provides a clear visualization of the relationship between cluster networks, network configurations, and VM networks, as well as how they impact VM scheduling on hosts.
```